source/upgrading/upgrade/_sysvm_templates.rst

```diff
@@ -48,9 +48,7 @@ Update System-VM templates
    |            |                                                            |
    |            | Format: VHD                                                |
    |            |                                                            |
-   |            | OS Type: Debian GNU/Linux 7.0 (64-bit) (or the             |
-   |            | highest Debian release number available in the             |
-   |            | dropdown)                                                  |
+   |            | OS Type: Other Linux 64-bit                                |
    |            |                                                            |
    |            | Extractable: no                                            |
    |            |                                                            |
```

Comment:
for al hypervisors we might want to add `Dynamically Scalable` and `HVM` to these lists, in order not to confuse users

Reply:
HVM doesn't show for some hypervisor afaik,  but yes valid point (have been asked questions about that recently)

Reply:
not needed, fixed the template OS type name.